src/toil_scripts/gatk_germline/germline.py

```diff
@@ -42,10 +42,11 @@
 
 import yaml
 from toil.job import Job
+from toil_lib import require
```

Comment:
PyCharm's `optimize imports` feature doesn't put a space there, but puts 1 line of whitespace between that block and toil_scripts.  Do you have a preference?